digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/lane/LaneService.scala

```diff
@@ -1262,7 +1262,9 @@ trait LaneOperations {
       val linkId = linkPropertyChange.roadLinkFetched.linkId
       val newValue = linkPropertyChange.linkProperty.linkType.value
       val oldValue = linkPropertyChange.optionalExistingValue.getOrElse(99)
-      val twoWayLaneLinkTypeChange = twoWayLanes.map(_.value).contains(newValue) || twoWayLanes.map(_.value).contains(oldValue)
+      val newValueIsTwoWayLane = twoWayLanes.map(_.value).contains(newValue)
+      val oldValueIsTwoWayLane =  twoWayLanes.map(_.value).contains(oldValue)
+      val twoWayLaneLinkTypeChange = newValueIsTwoWayLane ^ oldValueIsTwoWayLane
```

Comment:
mikä operaattori tää on "^"? heitätkö linkin missä selitetty.

Reply:
XOR https://www.geeksforgeeks.org/operators-in-scala/

Reply:
pittäs oman ymmärryksen mukaan toimia boolean arvojen kanssa tässä niin kuin halutaankin